extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/cdi/WithSpanInterceptor.java

```diff
@@ -22,11 +22,11 @@
 import io.opentelemetry.instrumentation.annotations.WithSpan;
 import io.opentelemetry.instrumentation.api.annotation.support.MethodSpanAttributesExtractor;
 import io.opentelemetry.instrumentation.api.annotation.support.ParameterAttributeNamesExtractor;
+import io.opentelemetry.instrumentation.api.incubator.semconv.util.SpanNames;
```

Comment:
can you open an issue about stabilizing this class?

Reply:
Sure! https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/11693